pages/index.js

```diff
@@ -3,17 +3,59 @@ import Footer from '../components/Footer';
 import Jumbotron from '../components/Jumbotron';
 import GameCard from '../components/GameCard';
 import SeeMore from '../components/SeeMore';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
 
 export default function Home() {
+
+  const [games, setGame] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(process.env.NEXT_PUBLIC_BASE_URL + "games/listing")
+      .then((res) => {
+        setGame(res.data?.data);
+        })
+      .catch((err) => {
+      console.log(err);
+      });
+  })
+
+  const handlePlay = () => {
+		axios
+		.post(process.env.NEXT_PUBLIC_BASE_URL + "games/view-count/" + id)
+		.then((res) => {
+			router.push("/games/detail-games/" + id);
+		})
+		.catch((err) => {
+			router.push("/games");
+			console.log(err);
+		});
+	};  
+
+
   return (
     <>
       <Navbar />
       <Jumbotron />
       <div className="text-3xl font-bold text-center mb-5 mt-8">Most Played Games</div>
       <div className="md:grid md:grid-cols-3 md:gap-8 px-10 py-4 xl:px-40 2xl:px-80">
-        <GameCard />
-        <GameCard />
-        <GameCard />
+        {
+          games.map((game) => {
```

Comment:
karna di landing page butuhnya 3 game card (sisanya di see more), kalo codenya begini berarti masih ngeakses semua data game ya mas? apa baiknya dibatasin jadi 3 game card aja atau gimana?

Reply:
Kalau saya tidak salah, ketika kita klik see more aksesnya ke list games yg menampilkan semua data game, Mas. 
*CMIIW.

Reply:
Nah iya bener mas, nanti dia harusnya ngelink ke /games/detail-games kalo klik see more. Tapi kalau seinget saya tampilan di home (/games) yg ditampilkan hanya 3 game card, sedangkan setau saya .map itu ngerunning ke semua isi array.. cuman saya jg kurang tau bener salahnya mas, mungkin coba tanya masukan dr temen2 yg lain ya🙏🏽

Reply:
pas nge GET games listing bisa make params buat dapet top x games mas

Reply:
pake /games/landing aja mas, ini aku mau adjust dikit di endpoint BE nya biar ngeluarin top 3 aja

Reply:
untuk yg ini mas, kalau dr BE bisa dibatasin jd 3 game card kah? atau mau dari FEnya dikasih slicing ketika mau mapping aja? jadi `games.slice(0, 3).map((game) => {...})`. nanti yang dr BE tinggal sorting arraynya dr view-count sama play-count terbanyak gt mas

Reply:
@enggarlr dari BE udah sorted kok mas

/games/landing dan /games/listing 2-2nya function yg sama, bedanya di landing resultnya dibatesin jadi 3